plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.ts

```diff
@@ -91,6 +94,10 @@ export class OktaGroupEntityProvider extends OktaEntityProvider {
     );
     this.orgUrl = accountConfig.orgUrl;
     this.groupFilter = accountConfig.groupFilter;
+    this.customAttributesToAnnotationAllowlist =
+      options.customAttributesToAnnotationAllowlist
+        ? options.customAttributesToAnnotationAllowlist
+        : [];
```

Comment:
we could write this expression like the below suggestion. I think it is easier to read.
```suggestion
    this.customAttributesToAnnotationAllowlist = options.customAttributesToAnnotationAllowlist || [];
```